### DISCUSSION TITLE
Access to Proxmox menu

### DISCUSSION THREAD

**Discussion by FredrikAndersson** [tteck/Proxmox#2564] [Help Needed] [answered]
Hi, I used Plex Media Server LXC to create my VM.
In Proxmox I can't find the Hardware menu for this VM.
How can I access this?
Grateful for answers.

**Comment by oOStroudyOo** [comment]
Unless it's changed recently.. the script creates an LXC not a VM.
There are differences between LXCs and VMs.

**Comment by FredrikAndersson** [comment, marked as answer]
[SOLVED] OK thanks for the help
